sdk/communication/communication-email/package.json

```diff
@@ -49,7 +49,7 @@
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
     "@azure/core-client": "^1.3.2",
-    "@azure/core-lro": "~2.5.2",
+    "@azure/core-lro": "~2.5.1",
```

Comment:
why is this needed?

Reply:
When we tried running the JS SDK pipeline to publish our SDK, we ran into the following error.

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2606056&view=logs&j=48dcb17b-e806-56fe-7107-3deafd21fd40&t=b4684a09-b10a-586a-ad28-8a3fd165ecc7

Reply:
@azure/lro latest version in npmjs is currently 1.5.1, see [here](https://www.npmjs.com/package/@azure/core-lro?activeTab=versions)

Reply:
Ah I see, may I suggest to not pin the version and instead use `^2.5.0`?